syntax = "proto3";

service Chat {
    rpc connect(ChatUser) returns (ChatUserConnected);
}

message ChatUser {
    string username = 1;
}

message ChatUserConnected {
    string userId = 1;
    string username = 2;
}
